docs/apps/gdal.md

```diff
@@ -87,13 +87,21 @@ gdalinfo /vsis3/<name_of_your_bucket>/<name_of_your_file>
 
 With large quantities of raster data (also in Allas), the most convenient method of accessing them might be [GDAL virtual rasters](../support/tutorials/gis/virtual-rasters.md). 
 
-## License and acknowledgement
+## License 
 
 GDAL is licensed under an [MIT/X style license](https://gdal.org/license.html)
 
+## Citation
+
+```GDAL/OGR contributors (2022). GDAL/OGR Geospatial Data Abstraction software Library. Open Source Geospatial Foundation. URL https://gdal.org, DOI: 10.5281/zenodo.5884351 ```
```

Comment:
```suggestion
```
GDAL/OGR contributors (2022). 
GDAL/OGR Geospatial Data Abstraction software Library. 
Open Source Geospatial Foundation. 
URL https://gdal.org
DOI: 10.5281/zenodo.5884351 
```
```

Reply:
As this is meant for copying, I would leave it as oneliner.